app/utils/strings.ts

```diff
@@ -6,5 +6,5 @@
  */
 export function removeAsterisksAndHashes(input: string): string {
   // Remove all asterisks and hash symbols
-  return input.replace(/[#]/g, "");
+  return input.replace(/[*#]/g, "");
```

Comment:
This would strip asterisks used at bullet points and bold/italic as well so just double checking we're ok with messing with user expectations who are familiar with markdown to disable those features. Also, out of curiosity what flavor of Markdown are we using that uses `**` for headers? I'm not sure I've seen that before.

Reply:
So we are sharing this API data with SF Service Finder and we can't regulate how data is formatted on the API side. We ideally should not have any bold or italics (because it looks kind of chaotic) but definitely no headers (because it's inaccessible). I'll double check w/ design about removing bold / italics etc, but this is how it looks while keeping them. It looks like it's not maintaining the line breaks well either so I will spend a little more time on this in AM.
<img width="711" alt="image" src="https://github.com/Exygy/askdarcel-web/assets/60076088/d9962825-da0a-41e1-b0de-36b32b2c112d">

Reply:
Sounds good re: checking with design. Then we can write the resulting expectations into our function unit specs that which becomes a human readable contract about the behavior!